.github/PULL_REQUEST_TEMPLATE

```diff
@@ -7,5 +7,6 @@ https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch
 ## Contributor Checklist:
 
 * [ ] There is an associated ticket in Trac. Create a new one at https://twistedmatrix.com/trac/newticket
+* [ ] The changes pass minimal style checks (see: https://twistedmatrix.com/trac/wiki/TwistedDevelopment#GettingYourPatchAccepted )
```

Comment:
You will update the example at this link to incorporate a `pycodestyle` env?

    $ tox -e pyflakes,newsfragment,manifest-checker

Reply:
yes... but I hope that soon we will have a newer and faster command... as the whole tox rig is slow :)